user/how-to-guides/how-to-open-urls-in-other-qubes.md

```diff
@@ -1,174 +1,105 @@
 ---
 lang: en
 layout: doc
-permalink: /doc/how-to-open-urls-in-other-qubes/
-ref: TBD
-title: How to open URLs/files in other qubes
+permalink: /doc/how-to-open-urls-and-files-in-other-qubes/
+title: How to open URLs and files in other qubes
 ---
 
-*This page is about opening untrusted files and URLs from "secure" offline or
-firewalled qubes in "general-purpose" qubes - or any qubes you see fit. The
-simplest solution is to [copy/paste](/doc/how-to-copy-and-paste-text/) URLs or
-[copy files](/doc/how-to-copy-and-move-files/) between qubes and manually open
-the resource in the destination qube. However this approach is error-prone so
-using formal RPC policies like described below is preferable.*
```

Comment:
I don't think it's quite accurate to say that this approach is "error-prone." There is nothing about the process itself that makes errors likely.

Reply:
> There is nothing about the process itself that makes errors likely.

I'm a heavy user of inter-vm copy/paste and I've pasted content in the wrong vm *many* times, including important passwords that I would then have to change - usually because of inattention but sometimes because a window from another vm would pop up at the same moment I was ctrl-shift-v'ing. That's what I meant with "error-prone" and "preferably" using an automated RPC policy.
(that's just a FYI - you did a very good job reformating/fixing the doc!).

Reply:
> sometimes because a window from another vm would pop up at the same moment I was ctrl-shift-v'ing. That's what I meant with "error-prone" and "preferably" using an automated RPC policy.

Ah, but that's not a problem with the global clipboard. That's a problem with your window manager settings allowing windows to pop up in ways you don't want. The exact same problem can and does occur on non-Qubes OSes with regular copy/paste (and perhaps even more commonly with mouse clicks).